go.mod

```diff
@@ -3,22 +3,23 @@ module github.com/dell/cert-csi
 go 1.22
 
 require (
-	github.com/cheggaaa/pb/v3 v3.0.2
-	github.com/dell/csm-replication v1.7.1
-	github.com/fatih/color v1.15.0
+	github.com/cheggaaa/pb/v3 v3.1.5
+	github.com/dell/csi-volumegroup-snapshotter v1.5.1
+	github.com/dell/csm-replication v1.8.0
```

Comment:
Last release of replication:

https://github.com/dell/csm-replication/releases/tag/v1.8.1 

Do we need to point out to this specific version?

Reply:
Sure let me update it